packages/module/patternfly-docs/content/examples/catalogTile.css

```diff
@@ -1,5 +1,5 @@
 .ws-react-e-catalog-tile .catalog-tile-pf.featured {
-  border-top: 2px solid #39a5dc;
+  border-top: 2px solid var(--pf-t--global--border--color--brand--default);
```

Comment:
The featured card can remove this top border. Looking at the designs it seems like the secondary card is used instead of a specific featured card.

Reply:
Ok, updated the variant to be secondary when it's featured!